.github/workflows/release.yml

```diff
@@ -47,7 +47,7 @@ jobs:
           token: ${{ secrets.DESIGN_REPO_ACCESS_TOKEN }}
           repository: mongodb/design
           event-type: release-leafygreen-ui
-          client-payload: '{"ref": "${{ github.ref }}", "sha": "${{ github.sha }}", "packages": "${{steps.changesets.outputs.publishedPackages}}"}'
+          client-payload: '${{ steps.changesets.outputs.publishedPackages }}'
```

Comment:
As long as we don't need those other properties, this should work?

Reply:
Thought I wonder if it's the format of the `publishedPackages` string that's the issue

Reply:
```
client-payload: [{ "name": "..." }]
```
Seems like the double-quotes may be causing the issue